Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -968,29 +977,28 @@ extension NavigationViewController: TopBannerViewControllerDelegate {
     }
 }
 
-// MARK: - BottomBannerViewControllerDelegate
-
-// Handling cancel action in new Bottom Banner container.
+// MARK: - BottomBannerViewControllerDelegate methods
 
 extension NavigationViewController: BottomBannerViewControllerDelegate {
     
+    // Handle cancel action in new Bottom Banner container.
     public func didTapCancel(_ sender: Any) {
         handleCancelAction()
     }
 }
 
-// MARK: - CarPlayConnectionObserver
+// MARK: - CarPlayConnectionObserver methods
```

Comment:
Haven't seen this addition in other places in our code, do we really need it?

Reply:
There are other marks in the codebase named “_Protocol name_ methods”. This is a holdover from Objective-C style. In Swift, it’s common to set off a class’s conformance of a particular protocol in its own extension, so marks become less important for organization.

Reply:
I personally use such markers while navigating through large files in top bar menu of Xcode.